src/AtmosphereModels/radiation_interface.jl

```diff
@@ -92,7 +104,7 @@ RadiativeTransferModel
 ```
 """
 function RadiativeTransferModel(grid::AbstractGrid, optics, args...; kw...)
-    msg = "Unknown optics $(optics). Valid options are GrayOptics(), ClearSkyOptics().\n" *
+    msg = "Unknown optics $(optics). Valid options are GrayOptics(), ClearSkyOptics(), AllSkyOptics().\n" *
```

Comment:
It's slightly unfortunate we need to list the types explicitly here, but the only solution I see is using `InteractiveUtils.subtypes()`, but I'd rather avoid adding a dependency on `InteractiveUtils` if possible (although I see `GPUCompiler`, `KernelAbstractions`, and `Oceananigans` already do it, so it wouldn't be too bad after all)